internal/ethapi/simulate_test.go

```diff
@@ -119,3 +121,18 @@ func TestSimulateSanitizeBlockOrder(t *testing.T) {
 func newInt(n int64) *hexutil.Big {
 	return (*hexutil.Big)(big.NewInt(n))
 }
+
+func TestHeaderPassing(t *testing.T) {
```

Comment:
I don't understand this test, maybe it was left by accident?

Reply:
Yes! removed